serverless.yml

```diff
@@ -1,13 +1,17 @@
 service: slowcovid
 app: slowcovid
-org: jeffreysilver
```

Comment:
This is the one piece of configuration I'm not sure we can remove in dev/prod (it might be fine, would probably just to deploy & see, which we could do by temporarily disabling the prd dialog-engine pipeline). It's necessary for deploying locally, though